pkg/api/controller.go

```diff
@@ -3854,6 +3854,78 @@ func (c *Controller) PostStatsEvents(w http.ResponseWriter, r *http.Request, bod
 	writeResponse(w, r, http.StatusNoContent, nil)
 }
 
+func (c *Controller) OtfDiff(w http.ResponseWriter, r *http.Request, repository string, leftRef string, rightRef string, params OtfDiffParams) {
+	ctx := r.Context()
+	user, err := auth.GetUser(ctx)
+	if err != nil {
+		writeError(w, r, http.StatusUnauthorized, ErrAuthenticatingRequest)
+		return
+	}
+	c.LogAction(ctx, "table_format_diff", r, repository, rightRef, leftRef)
```

Comment:
I would add params.Type to the event name. i.e. "table_format_diff_" +params.Type